docs/pipeline/security/customer-managed-keys/index.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Customer managed keys"
+sidebar_position: 1
+---
+
+This guide provides step-by-step instructions for setting up AWS KMS (Key Management Service) custom keys for your Snowplow pipeline. Custom KMS keys provide enhanced security by allowing you to maintain full control over encryption keys used by your data infrastructure.
+
+:::note
+
+This is a bolt-on security feature available for enterprise customers using BDP PMC (Private Managed Cloud) or BDP Cloud deployments.
+
+:::
+
+AWS Custom KMS Keys ensure that all data stored at-rest in AWS services (Kinesis, SQS, and S3) are encrypted with a key that you own and control. This provides:
+
+- Enhanced security: full control over encryption keys
+- Compliance: meet regulatory requirements for data encryption
+- Audit control: ability to revoke access to encrypted data
+- Key management: centralized control over data encryption
+
+## Prerequisites
+
+To use custom KMS keys, you will need:
+- AWS account with appropriate permissions to create and manage KMS keys
+- Access to your AWS Console in the same region as your Snowplow pipeline
+- Snowplow pipeline account ID (provided by your Customer Success Manager)
+- Security team approval for cross-account key sharing
+
+This feature has some limitations:
+- EMR shredder: if you're using the legacy EMR shredder process, it may not fully respect custom KMS keys for all output. This system is being deprecated.
```

Comment:
Is this what is now known as RDB Transformer? (The non-streaming kind.) In that case, it’s not being deprecated for Redshift. So maybe we need to rephrase this to say that when loading to Redshift, keys might not be respected (what does that mean?).